v2/slogger/logger.go

```diff
@@ -82,8 +93,9 @@ func getSizeInKb(n int) string {
 func getTruncatedMessage(old string) string {
 	new := old
 	lineLen := len(old)
-	if maxLogSize > MinimumMaxLogSizeThreshold && lineLen > maxLogSize+4 {
-		new = fmt.Sprintf("%s...%s (warning: log line attempted (%vk) over max size (%vk), printing beginning and end)", old[0:maxLogSize], old[lineLen+3-MinimumMaxLogSizeThreshold:], getSizeInKb(lineLen), getSizeInKb(maxLogSize))
+	mls := getMaxLogSize()
+	if mls > MinimumMaxLogSizeThreshold && lineLen > mls+4 {
+		new = fmt.Sprintf("%s...%s (warning: log line attempted (%vk) over max size (%vk), printing beginning and end)", old[0:mls], old[lineLen+3-MinimumMaxLogSizeThreshold:], getSizeInKb(lineLen), getSizeInKb(maxLogSize))
```

Comment:
Does this comment need to be updated to use mls instead of maxLogSize?

Reply:
oops! fixed!